.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @npmccallum
```

```suggestion
*       @npmccallum @haraldh 
```

---

```suggestion
*       @npmccallum @haraldh @connorkuehl 
```

@connorkuehl You've earned a seat at the table.